<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>424</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>496</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Đăng nhập</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none; 
	border-radius: 5px;
	color: white;
	background: rgb(0, 85, 255);
}

QPushButton:hover { 
	background: rgba(0, 85, 255, 200);
}

QPushButton:pressed { 
	background: rgba(0, 85, 255, 180);
}</string>
    </property>
    <property name="text">
     <string>ĐĂNG NHẬP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_lt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: none; 
	padding: 5px; 
	border-bottom: 1px solid #ccc;
}

QLineEdit:focus {
	border-bottom: 1px solid rgba(0, 85, 255, 180);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Tên đăng nhập</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="host_lt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: none; 
	padding: 5px; 
	border-bottom: 1px solid #ccc;
}

QLineEdit:focus {
	border-bottom: 1px solid rgba(0, 85, 255, 180);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_lt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: none; 
	padding: 5px; 
	border-bottom: 1px solid #ccc;
}

QLineEdit:focus {
	border-bottom: 1px solid rgba(0, 85, 255, 180);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Mật khẩu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ĐĂNG NHẬP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="database_lt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: none; 
	padding: 5px; 
	border-bottom: 1px solid #ccc;
}

QLineEdit:focus {
	border-bottom: 1px solid rgba(0, 85, 255, 180);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="version_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Version:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>host_lt</tabstop>
  <tabstop>database_lt</tabstop>
  <tabstop>username_lt</tabstop>
  <tabstop>password_lt</tabstop>
  <tabstop>login_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
